gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -1662,9 +1661,17 @@ gst_tensor_buffer_append_memory (GstBuffer * buffer, GstMemory * memory,
     return FALSE;
   }
 
-  if (!gst_tensor_info_validate (info)) {
-    nns_loge ("Failed to get tensor info (invalid input info).");
-    return FALSE;
+  if (gst_tensor_meta_info_parse_memory (&meta, memory)) {
+    is_static = (meta.format == _NNS_TENSOR_FORMAT_STATIC);
```

Comment:
Is it nice to create function ```gst_tensor_meta_info_get_format``` later?

Reply:
I am not sure, we dont have any function to get the values in GstTensorMetaInfo because GstTensorMetaInfo struct is public.

Reply:
Okay, Thank you.